frontend/src/styles.scss

```diff
@@ -1,5 +1,9 @@
 @use './theme.scss' as theme;
```

Comment:
@jculvey I've renamed this file from `global` to `styles` as it looked simpler with the new folder structure (without a `styles` folder), let me know if you prefer `global` or anything else, I'm fine with whatever name you'd prefer 🙂

Reply:
That works. The `global` bit is just a hint that these are styles that are applied globally `* { ... }, body: { ... }` etc. Now that we're using tailwind 100% there's only this file at the top level so the distinction isn't really required anymore